### PR TITLE
No more ping and remove ping settings

### PR DESCRIPTION
**Pull request by Kumamon38**

### CODE REVIEW

File: include/nakama-cpp/realtime/NRtTransportInterface.h [hunk 1]
```diff
@@ -63,14 +49,14 @@ namespace Nakama {
...
          */
-        virtual void setPingSettings(const NRtPingSettings& settings) = 0;
+        virtual void setActivityTimeout(uint16_t timeoutSec) = 0;
 
         /**
-         * Get ping settings.
+         * Get activity timeout.
          */
-        virtual NRtPingSettings getPingSettings() const = 0;
+        virtual uint16_t getActivityTimeout() = 0;
```
Comment (Dimon4eg) [inline include/nakama-cpp/realtime/NRtTransportInterface.h]: add `const`

File: src/realtime/NWebsocketCppRest.cpp [hunk 2]
```diff
@@ -28,8 +28,7 @@
 namespace Nakama {
 
 NWebsocketCppRest::NWebsocketCppRest():
-_lastSentPingTimeMs(0),
-_lastReceivedPongTimeMs(0)
+_lastReceivedMessageTimeMs(0)
```
Comment (Dimon4eg) [inline src/realtime/NWebsocketCppRest.cpp]: in C++11 you can initialize class members in header

Comment (Kumamon38) [reply src/realtime/NWebsocketCppRest.cpp]: Should we? Here it was hidden

Comment (Dimon4eg) [reply src/realtime/NWebsocketCppRest.cpp]: for `atomic` no initialization needed: https://en.cppreference.com/w/cpp/atomic/atomic/atomic

Comment (Kumamon38) [reply src/realtime/NWebsocketCppRest.cpp]: we have to let it here as this is an atomic that can't be initialize in the header

**Pull request merged**